virttest/utils_net.py

```diff
@@ -4247,3 +4247,51 @@ def get_netkvm_param_value(vm, param):
         return value
     finally:
         session.close()
+
+
+def create_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
+    """
+    Create ovs bridge via tmux command on local or remote
+
+    :param ovs_bridge_name: The ovs bridge
+    :param session: The remote session
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: The command status and output
+    """
+    runner = local_runner
+    if session:
+        runner = session.cmd
+    iface_name = get_net_if(runner=runner, state="UP")[0]
+    if not utils_package.package_install("tmux", session):
+        raise exceptions.TestError("Failed to install the tmux packages.")
```

Comment:
we may also need to check the openvswitch package install status, please also check the func setup_ovs_vhostuser and in it there are checks, I think maybe you can reuse or make them a separate func to avoid the code duplication

Reply:
Thanks @kylazhang ! Updated.